packages/duckietown_msgs/msg/DiagnosticsRosTopic.msg

```diff
@@ -0,0 +1,28 @@
+# Topic direction (this has to match duckietown.TopicDirection)
```

Comment:
Isn't the same topic inbound and outbound for at least 2 nodes? Or we publish one of these for every pair (node, topic)?